src/celeritas/em/distribution/WentzelDistribution.hh

```diff
@@ -0,0 +1,341 @@
+    if (target_.atomic_number().get() > 1)
+    {
+        real_type tau = inc_energy_ / inc_mass_;
+        // TODO: Reference for this factor?
+        real_type factor = sqrt(tau / (tau + sq_cbrt_z));
+        real_type inv_beta_sq = 1 + ipow<2>(inc_mass_) / inc_mom_sq();
+
+        correction = min(target_.atomic_number().get() * real_type{1.13},
+                         real_type{1.13}
+                             + real_type{3.76}
+                                   * ipow<2>(target_.atomic_number().get()
+                                             * constants::alpha_fine_structure)
+                                   * inv_beta_sq * factor);
+    }
+
+    return correction * data_.screening_factor * screen_r_sq_elec() * sq_cbrt_z
+           / inc_mom_sq();
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Calculate the (cosine of the) maximum polar angle that incident particle
+ * can scatter off of the target's electrons.
+ */
+CELER_FUNCTION real_type WentzelDistribution::compute_max_electron_cos_t() const
+{
+    // TODO: Need to validate against Geant4 results
+    real_type max_energy = is_electron_ ? real_type{0.5} * inc_energy_
+                                        : inc_energy_;
+    real_type final_energy = inc_energy_ - min(cutoff_energy_, max_energy);
+
+    if (final_energy > 0)
+    {
+        real_type incident_ratio = 1 + 2 * inc_mass_ / inc_energy_;
+        real_type final_ratio = 1 + 2 * inc_mass_ / final_energy;
+        real_type cos_t_max = sqrt(incident_ratio / final_ratio);
+
+        return clamp(cos_t_max, real_type{0}, real_type{1});
+    }
+    else
+    {
+        return 0;
+    }
+}
+//---------------------------------------------------------------------------//
+/*!
+ * Calculate the momentum scale for the nuclear form factor. This should
+ * be a constant function but it has a special case for hydrogen.
+ */
+CELER_FUNCTION real_type WentzelDistribution::nuclear_form_momentum_scale() const
+{
+    // TODO: Geant has a different form momentum scale for hydrogen?
+    if (target_.atomic_number().get() == 1)
+    {
+        return 1 / real_type{3.097e-6};
+    }
+    return native_value_to<MomentumSq>(
+               12
+               * ipow<2>(2 * constants::hbar_planck
+                         / (real_type(1.27e-15) * units::meter)))
+        .value();
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Calculate the screening R^2 coefficient for incident electrons. This is
+ * the constant prefactor of [PRM] eqn 8.51
+ */
+CELER_CONSTEXPR_FUNCTION real_type WentzelDistribution::screen_r_sq_elec() const
+{
+    //! Thomas-Fermi constant C_TF
+    //! \f$ \frac{1}{2}\left(\frac{3\pi}{4}\right)^{2/3} \f$
+    constexpr real_type ctf = 0.8853413770001135;
+
+    return native_value_to<MomentumSq>(
+               ipow<2>(constants::hbar_planck / (2 * ctf * constants::a0_bohr)))
+        .value();
```

Comment:
When `inc_mass_` is the electron mass, they work out to be equal. Geant differs by a factor of 2 because their `screenZ` is actually `2A` where `A` is the screening coefficient. Since I simplified the cross sections for just electrons and use just `A` in places, I figured it'd be clearer to have that distinction. My math is in section 4 of the overleaf document I'll comment below.